internal/services/containerapps/container_app_resource_test.go

```diff
@@ -264,6 +264,21 @@ func TestAccContainerAppResource_completeUpdate(t *testing.T) {
 	})
 }
 
+func TestAccContainerAppResource_completeTcpExposedPort(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_container_app", "test")
+	r := ContainerAppResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data, "rev1"),
```

Comment:
This should probably be using the test config added in this PR 
```suggestion
			Config: r.completeTcpExposedPort(data, "rev1"),
```

Reply:
Nice catch

Reply:
Unfortunately we have a test failure @SpartakusMd 
```
------- Stdout: -------
=== RUN   TestAccContainerAppResource_completeTcpExposedPort
=== PAUSE TestAccContainerAppResource_completeTcpExposedPort
=== CONT  TestAccContainerAppResource_completeTcpExposedPort
    testcase.go:113: Step 1/2 error: Error running apply: exit status 1
        Error: creating Container App (Subscription: "*******"
        Resource Group Name: "acctestRG-CAEnv-231101113719484045"
        Container App Name: "acctest-capp-231101113719484045"): performing CreateOrUpdate: containerapps.ContainerAppsClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: Code="ContainerAppInvalidIngressAllowInsecureForTcpApp" Message="Allow insecure can only be set for http transport."
          with azurerm_container_app.test,
          on terraform_plugin_test.tf line 130, in resource "azurerm_container_app" "test":
         130: resource "azurerm_container_app" "test" {
        creating Container App (Subscription: "*******"
        Resource Group Name: "acctestRG-CAEnv-231101113719484045"
        Container App Name: "acctest-capp-231101113719484045"): performing
        CreateOrUpdate: containerapps.ContainerAppsClient#CreateOrUpdate: Failure
        sending request: StatusCode=0 -- Original Error:
        Code="ContainerAppInvalidIngressAllowInsecureForTcpApp" Message="Allow
        insecure can only be set for http transport."
--- FAIL: TestAccContainerAppResource_completeTcpExposedPort (514.69s)
FAIL
```